src/config.c

```
@@ -275,11 +275,13 @@ bool load_configuration(const char *override_configpath, config_load_t load_type
 
     /* Make bar config blocks without a configured font use the i3-wide font. */
     Barconfig *current;
-    TAILQ_FOREACH (current, &barconfigs, configs) {
-        if (current->font != NULL) {
-            continue;
+    if (load_type != C_VALIDATE) {
```

I wonder if the check should be config.font != NULL to be extra safe

---

Reason I didn't do that is because `load_font` should exit if it fails to load any fallback fonts, so that check might hide some other crashing failure.